lib/pbench/server/utils.py

```diff
@@ -73,6 +75,11 @@ def quarantine(dest, logger, *files):
         if not os.path.exists(afile) and not os.path.islink(afile):
             continue
         try:
+            if str(afile).endswith(".tar.xz"):
```

Comment:
This is the second place where the code has explicitly referenced `".tar.xz"`...should this be encapsulated in a symbolic reference or, better yet, abstracted into a function?

Reply:
This is a widespread assumption throughout the code. Yes, it really should be in `constants.py` and referenced consistently. Much of the code was converted from `bash` ... although that's not really an excuse. I know I mistyped this in one place as `.tar.xy` and scratched my head for minutes before I realized why it wasn't working. A central constant would alleviate at least some of that sort of trouble. Nobody's gotten around to it!